ruby/sinatra/app.rb

```diff
@@ -0,0 +1,77 @@
+
+require 'bundler/setup'
+
+require 'connection_pool'
+require 'date'
+require 'json'
+require 'redis'
+require 'sinatra'
+require 'sinatra/json'
+
+require 'ddtrace'
+require 'ddtrace/contrib/sinatra/tracer'
+
+require './post.rb'
+
+class App < Sinatra::Application
+  REDIS_HOST = '127.0.0.1'.freeze()
```

Comment:
use ENV vars with defaults so that it can be deployed easily in nodes that don't have the service installed in ``localhost`` (i.e. docker deployments).